api/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+pub mod context;
+pub mod filters;
+pub mod handlers;
+pub mod runtime;
+
+#[cfg(any(test))]
+pub(crate) mod tests;
```

Comment:
Why have `tests/mod.rs` and make this public if it's never consumed in this PR? As opposed to have the test_get_ledger_info  tests in the same file as filters.

Reply:
Because we are going to have a lot of tests, and we may also need separate test utils.
As example, please see unit_tests file in the json-rpc/src/tests dir.

Reply:
We will also put all endpoint filters / handlers into the filters.rs and handlers.rs files, so they will be pretty large anyway.
From test perspective, there is almost no difference to have tests in the same file or put it into separate file/mod